integration-examples/apiScripts/token.yaml

```diff
@@ -1,2 +1,6 @@
-access_token: Splunk-O11y-User-API-Access-Token
-realm: us1
\ No newline at end of file
+access_token:  #required. User API Access Token from your Splunk Observability Cloud Organization
+realm:  #required. Realm for your Splunk Observability Cloud Organization (ex: us1)
+emailAddress:   # required. Email address that you would like to add to the detector(s)
```

Comment:
Is it worth detailing these fields and comments related to them in the README.md for documentation? 
That was users know right off what they need to add to get going.

Reply:
Yes, great call out!! Done